src/api.js

```diff
@@ -2360,8 +2360,8 @@ api.declare({
   description: [
     'Get a list of all active workers of a workerType.',
     '',
-    '`listWorkers` allows a response to be filtered by the `disabled` property.',
-    'To filter the query, you should call the end-point with `disabled` as a query-string option.',
+    '`listWorkers` allows a response to be filtered by the `quarantineUntil` property.',
+    'To filter the query, you should call the end-point with `quarantineUntil` as a query-string option.',
```

Comment:
What does this return?  Those quarantined before that date? after?

Reply:
after